src/admin/migrations/m200706_202002_cms_website.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Class m200706_202002_cms_website
+ */
+class m200706_202002_cms_website extends Migration
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function safeUp()
+    {
+        $this->createTable('cms_website',[
+            'id' => $this->primaryKey(),
+            'name' => $this->string(20)->unique()->notNull(),
+            'is_default' => $this->boolean()->notNull()->defaultValue(false),
+            'is_active' => $this->boolean()->notNull()->defaultValue(false),
+            'is_deleted' => $this->boolean()->notNull()->defaultValue(false),
+            'host' => $this->string(255)->unique()->notNull(),
```

Comment:
unique index can only be `190` chars length, use `string(190)`` - notNull is not needed

Reply:
ahh ok, interesting the char limit with utf8mb4. Good to know! 
But it should be not nullable because otherwise it could be multiple entries with a null value despite the unique index.

Reply:
ok